ui/src/duck/employee.ts

```diff
@@ -47,8 +47,12 @@ export default createSlice({
     });
 
     builder.addCase(fetchCurrentUser.fulfilled, (state, { payload }) => {
-      state.employees = payload.employees;
-    });  },
+      const allEmployee = [...state.employees, ...payload.employees];
+      state.employees =  allEmployee.filter((item, pos) => {
+        return allEmployee.indexOf(item) === pos;
```

Comment:
Please test this with multiple employees. It is possible that immer would provide referential integrity here, but I am not sure about that. Please verify.